cooperatives/open-path.md

```diff
@@ -21,6 +21,10 @@ parent: Cooperatives
 
 Open Path is a suite of open source software for governments’ Coordinated Access System (CAS) and [Homeless Management Information Systems](https://www.hudexchange.info/programs/hmis/) (HMIS), used to collect client-level data and data on the provision of housing and services to homeless individuals and families and persons at risk of homelessness. They’re two complementary tools that are jointly titled “Open Path.”
 
-Open Path was created when [Boston's Department of Neighborhood Development hired Green River](https://www.greenriver.com/portfolio/the-city-of-boston), a Vermont-based software development vendor, to create components of a CAS and HMIS for them. Boston released the resulting software under an open source license, allowing it to be reused freely. [Green River now sells SaaS hosting for Open Path](https://www.openpath.host/how-to-get-it), although any government is also free to use Open Path without hiring Green River.
+Open Path was created when [Boston's Department of Neighborhood Development hired Green River](https://www.greenriver.com/portfolio/the-city-of-boston), a Vermont-based software development vendor, to create components of a CAS and HMIS for them. Boston released the resulting software under an open source license, allowing it to be reused freely. Boston came up with the data driven performance evaluation. Fort Worth seized it after Boston presented, and the changes they made, made it back to Boston. [Green River now sells SaaS hosting for Open Path](https://www.openpath.host/how-to-get-it), although any government is also free to use Open Path without hiring Green River.
```

Comment:
This sentence is unclear: "Fort Worth seized it after Boston presented, and the changes they made, made it back to Boston." Can you take a stab at rephrasing? @abbeyns149